spec/system/searching_spec.rb

```diff
@@ -194,6 +194,7 @@
     end
 
     it 'can add a facet to an existing search', js: true do
+      allow(Bibdata).to receive(:get_patron).and_return({ "patron_group" => "P" })
```

Comment:
Why is this mock needed?  I wouldn't expect the patron group to be checked just to do an advanced search.

Reply:
@rladdusaw I have the same question

Reply:
That test is a little flakey and sometimes fails for me locally. I guess I got caught up in adding that line to all the failing tests toward the end. I've removed it.